packages/lesmis/package.json

```diff
@@ -10,23 +10,21 @@
   },
   "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
   "main": "dist/index.js",
-  "scripts": {
-    "build": "npm run --prefix ../../ package:build:react-scripts",
+  "scripts": { 
+    "build": "cross-env NODE_OPTIONS=--max-old-space-size=8192 npm run --prefix ../../ package:build:vite",
```

Comment:
This is a workout for a [known issue with Vite](https://github.com/vitejs/vite/issues/2433)

Reply:
A bit of a concern but well found